src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -488,6 +541,12 @@ def post_process_segmentation(
             `torch.Tensor`:
                 A tensor of shape (`batch_size, num_class_labels, height, width`).
         """
+        logger.warning(
+            "`post_process_segmentation` is deprecated and will be removed in v5 of Transformers, please use"
```

Comment:
I don't think `post_process_segmentation` is a public method actually. It is not present in the docs, nor was it shown in any notebook.

So I think we can directly remove it?

Reply:
@NielsRogge @sgugger It is used within ImageSegmentationPipeline but I'm simultaneously working on a PR to fix the pipeline. I'll open the other PR once this PR is merged.

I will go ahead and remove `post_process_segmentation` though.

Reply:
SGTM!

Reply:
`post_process_segmentation` was included in the model docs so I couldn't remove it.